src/gui/shader.cpp

```diff
@@ -64,7 +69,10 @@ load_shaders(
     int info_log_length;
 
     // Compile Vertex Shader
-    LOG_DEBUG(logger, "Compiling vertex shader {}", vertex_file_path);
+    LOG_DEBUG(
+        logging::opengl_logger, "Compiling vertex shader {}",
+        vertex_file_path.lexically_normal().string()
```

Comment:
Already garunteed to be lexically normal in `utils/files.hpp`

Also don't need the cast to string.